corehq/apps/userreports/filters/__init__.py

```diff
@@ -11,6 +14,9 @@ class Filter(object):
     def __call__(self, item, context=None):
         return True
 
+    def __str__(self):
```

Comment:
I added this method on the main filter class but wondering why we don't combine the Filter object and the FilterSpec objects and make the filter specs callable (similar to expressions).